evaluate_segmentation.py

```diff
@@ -124,7 +124,9 @@ def main(params):
     gpkg_name_ = []
 
     for aoi in tqdm(list_aois, desc='Evaluating from input list', position=0, leave=True):
-        output_path = working_folder / f"{aoi.aoi_id}_pred.tif" if not inference_image else inference_image
+        print(f"\nEvaluating AOI: {aoi.aoi_id}")
```

Comment:
do we want to switch to logging,info() instead of print()?

Reply:
will take out the debug print statements. it's not meant to be there!